api/images/services/logo_detection_service.py

```diff
@@ -0,0 +1,19 @@
+from ml.models.logo_detection.process import process as detect_logos
+
+# from rest_framework import serializers
+
+
+class logo_detection_service:
```

Comment:
Sorry probably wasn't clear enough lol, class names should be `UpperCamelCase` and then file names, variable names and method / function names should be `snake_case`
```suggestion
class LogoDetectionService:
```